crates/mdk-core/src/encrypted_media/manager.rs

```diff
@@ -14,7 +14,7 @@ use crate::encrypted_media::metadata::extract_and_process_metadata;
 use crate::encrypted_media::types::{
     EncryptedMediaError, EncryptedMediaUpload, MediaProcessingOptions, MediaReference,
 };
-use crate::encrypted_media::validation::{validate_filename, validate_inputs, validate_mime_type};
+use crate::image_processing::validation;
```

Comment:
These validations are made for not only images, but also other media types. Should we rename it to media_processing maybe?